OpenOversight/app/main/downloads.py

```diff
@@ -160,8 +160,5 @@ def descriptions_record_maker(description: Description) -> _Record:
     return {
         "id": description.id,
         "text_contents": description.text_contents,
-        "created_by": description.created_by,
         "officer_id": description.officer_id,
-        "created_at": description.created_at,
-        "last_updated_at": description.last_updated_at,
```

Comment:
Unless there's something I'm missing, these fields shouldn't be needed. They aren't included in any other `_record_maker` functions, so my guess is that we do not need to include them here.